src/state/vesting/reducer.ts

```diff
@@ -57,7 +57,7 @@ export default createReducer<VestingState>(initialState, builder =>
           cause: error,
         })
         e.name = 'VestingError'
-        captureException(e, { level: Severity.Error })
+        captureException(e, { level: 'error' })
```

Comment:
Why, did they deprecated this?

Reply:
exactly

Reply:
```
/**
 * @deprecated Please use a `SeverityLevel` string instead of the `Severity` enum. Acceptable values are 'fatal',
 * 'error', 'warning', 'log', 'info', and 'debug'.
 */
export declare enum Severity {
    /** JSDoc */
    Fatal = "fatal",
    /** JSDoc */
    Error = "error",
    /** JSDoc */
    Warning = "warning",
    /** JSDoc */
    Log = "log",
    /** JSDoc */
    Info = "info",
    /** JSDoc */
    Debug = "debug"
}
export declare type SeverityLevel = 'fatal' | 'error' | 'warning' | 'log' | 'info' | 'debug';
//# sourceMappingURL=severity.d.ts.map
```